### ISSUE TITLE
User DB

### ISSUE THREAD

**Issue by droud-dev** [jaspreet338/tech-pandas-STAR#9]
User DB needs to store the information that's relevant to user authentication and profiles

Required fields:
- id (Comes from the GitHub API user record - id property)
- username (Comes from the GitHub API user record - login property)
- name (Initialized from the GitHub API user record - name property)
- area
- role (TA/student/mentor)
- class
- auth_token (This comes from the GitHub Auth API)

**Comment by MarkFarmiloe** [comment]
@droud-dev  I don't think we need to save the auth_token as we only use it at login to get the user details.
During the rest of the session, we can retrieve the user details from the database using the id (I suggest we use the Github id).
